.travis.yml

```diff
@@ -125,7 +125,8 @@ jobs:
     - stage: test
       env: NAME="esp32 port build"
       install:
-        - sudo python3 -m pip install pyparsing
+        - sudo apt-get install python3-pip
+        - sudo pip3 install pyparsing
```

Comment:
You may want to upgrade pip too, I recall having trouble with the distribution provided release of `python3-pip`.
```
apt-get install -y --no-install-recommends python3-pip
python3 -m pip install --upgrade pip setuptools
```